suisa_sendemeldung/suisa_sendemeldung.py

```diff
@@ -32,50 +32,47 @@
 
 Im Anhang finden Sie die Sendemeldung von $station_name für den $month $year.
 
-Wir senden ihnen diese Sendemeldung da wir bis Ende $previous_year nicht informiert
-wurden, dass $station_name im laufenden Jahr von der Meldepflicht befreit ist.
+Diese Sendemeldung erfolgt gemäss den Bestimmungen "Gemeinsamer Tarif S 2020 - 2024" [1] unter
```

Comment:
```suggestion
Diese Sendemeldung erfolgt gemäss den Bestimmungen "Gemeinsamer Tarif S 2020 – 2022" [1] unter
```

I don't think we should update the link `[1]` to [this](https://www.eschk.admin.ch/dam/eschk/de/data/beschluesse/2019/gt-s-2019.pdf.download.pdf/gt-s-2019.pdf) but we could possibly want to call the document by what was agreed upon in the political arena.

Reply:
@paraenggu het gseit es ischem in ändeffekt echli glych